bigbluebutton-html5/imports/ui/components/app/app-graphql/queries.ts

```diff
@@ -0,0 +1,14 @@
+import { gql } from '@apollo/client';
+
+export const GET_CURR_PRESS_PAGE_INFO = gql`
+  subscription MySubscription {
+    pres_page_curr {
+      fitToWidth
+      pageId
+    }
+  }
+`;
```

Comment:
Is it really necessary a new subscription dedicated for this prop?

Reply:
> Reference

I sent a fix for it